homeassistant/components/frontier_silicon/media_player.py

```diff
@@ -266,16 +284,19 @@ def is_volume_muted(self):
     async def async_volume_up(self):
         """Send volume up command."""
         volume = await self.fs_device.get_volume()
-        await self.fs_device.set_volume(volume + 1)
+        volume = int(volume or 0) + 1
+        await self.fs_device.set_volume(min(volume, self._volume_steps))
```

Comment:
What do you think about renaming `self._volume_steps` to `self._max_volume`?
Since it contains the value for the highest volume and not the step size in volume (range: 0-40, step size: 1).

Reply:
I agree, I had already considered renaming it but had left it purely to match the underlying API.  I'll rename it now.